daskernetes/core.py

```diff
@@ -228,10 +229,23 @@ def scale_up(self, n, **kwargs):
         """
         pods = self.pods()
 
-        out = [
-            self.core_api.create_namespaced_pod(self.namespace, self.pod_template)
-            for _ in range(n - len(pods))
-        ]
+        for i in range(3):
+            try:
+                out = [
+                    self.core_api.create_namespaced_pod(self.namespace, self.pod_template)
+                    for _ in range(n - len(pods))
+                ]
+                break
+            except client.rest.ApiException as e:
+                if 'ServerTimeout' in str(e):
+                    logger.info("Server timeout, retry #%d", i + 1)
+                    time.sleep(1)
+                    last_exception = e
+                    continue
+                else:
+                    raise
```

Comment:
I had odd API failures previously that I associated to running under a very recently created namespace.  This *may* have fixed things, but I'm not very confident here.  cc @yuvipanda

Reply:
Yeah, namespaces are 'initializing' for a short while, and reject objects inside it. This should be fine.